### PR TITLE
More Text Styles lesson: Updated font-style subsection

### LINKED ISSUE #23762: More Text Styles lesson: update font-style subsection
Lesson: [More Text Styles](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css/lessons/more-text-styles)

**1. Description:**

The [font-style subsection](https://github.com/TheOdinProject/curriculum/blob/main/html_css/intermediate_css/more-text-styles.md?plain=1#L70) can be improved. It should be split into two paragraphs and it should link to the [emphasis element mdn doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em). It should also have a second code example for the `em` tag.

**2. Acceptance Criteria:**

- [ ] Split current paragraph of font-style subsection into two. It could be split at "For example, if you want all your hea... ".
- [ ] The second paragraph should have an ending sentence along the lines of "The MDN doc on Emphasis Element puts stress on our point above". This sentence should also link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em
- [ ] The current code snippet should have a describing text: "We should use `font-style: italic;` if italics is required for styling purposes."
- [ ] Add a new code snippet on a use case for the `em` element. It should have the describing text: "We should use the `em` element if italics is required for emphasis."

**3. Additional Information:**

The above acceptance criteria is flexible. It can be modified with a better implementation. If anyone wants to work on this issue, comment to be assigned. In case of any questions, please feel free to ask.

### PR DESCRIPTION
**Pull request by ItsKatato**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #23762

**2. This PR:**

 - [x] Split current paragraph of font-style subsection into two. It could be split at "For example, if you want all your hea... ".
 - [x] The second paragraph should have an ending sentence along the lines of "The MDN doc on Emphasis Element puts stress on our point above". This sentence should also link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em
 - [x] The current code snippet should have a describing text: "We should use font-style: italic; if italics is required for styling purposes."
 - [x] Add a new code snippet on a use case for the em element. It should have the describing text: "We should use the em element if italics is required for emphasis."

### CODE REVIEW

File: html_css/intermediate_css/more-text-styles.md [hunk 1]
```diff
@@ -68,13 +68,23 @@ This method _may_ be more reliable than relying on an third-party font API, but
...
+Typically used to make a font italic. You learned about the HTML `<em>` tag, which uses an italic font, but `<em>` _also_ signifies that the text it wraps is significant or should be emphasized in some way. A good rule of thumb to follow is that if you just want text to be italic (or bold, underlined, highlighted, etc.), use a CSS property. Otherwise, if text should have some sort of semantic emphasis, use the correct HTML element.
+
+For example, if you want all your header text to be italic you should use `font-style` to accomplish this. If you want some text in the _middle of a sentence_ to appear italic in order to emphasize that text, it is appropriate to use an `em` element. The [MDN doc on the Emphasis Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em) puts stress on our point above.
 
 ~~~css
 h1 {
   font-style: italic;
 }
 ~~~
+We should use `font-style: italic;` if italics is required for styling purposes.
```
Comment (01zulfi) [inline html_css/intermediate_css/more-text-styles.md:80]: Can we have this sentence before the code snippet it's describing?

File: html_css/intermediate_css/more-text-styles.md [hunk 2]
```diff
@@ -68,13 +68,23 @@ This method _may_ be more reliable than relying on an third-party font API, but
...
+
+~~~html
+<p>I <em>never</em> said he stole your money</p>
+<p>I never said <em>he</em> stole your money</p>
+<p>I never said he stole <em>your</em> money</p>
+~~~
+We should use the `em` element if italics is required for emphasis.
```
Comment (01zulfi) [inline html_css/intermediate_css/more-text-styles.md:87]: Same here as well.

**Review by 01zulfi — CHANGES_REQUESTED** [review]
The changes look great and loving the example for `em`.

Couple of changes below, then we're good to merge! 🔥

File: html_css/intermediate_css/more-text-styles.md (hunk 1 above)
Comment (ItsKatato) [reply html_css/intermediate_css/more-text-styles.md]: I was thinking about that, too! Thanks for telling me, fixing it right now. 😅

File: html_css/intermediate_css/more-text-styles.md [hunk 3]
```diff
@@ -68,14 +68,24 @@ This method _may_ be more reliable than relying on an third-party font API, but
...
+Typically used to make a font italic. You learned about the HTML `<em>` tag, which uses an italic font, but `<em>` _also_ signifies that the text it wraps is significant or should be emphasized in some way. A good rule of thumb to follow is that if you just want text to be italic (or bold, underlined, highlighted, etc.), use a CSS property. Otherwise, if text should have some sort of semantic emphasis, use the correct HTML element.
 
+For example, if you want all your header text to be italic you should use `font-style` to accomplish this. If you want some text in the _middle of a sentence_ to appear italic in order to emphasize that text, it is appropriate to use an `em` element. The [MDN doc on the Emphasis Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em) puts stress on our point above.
+
+We should use `font-style: italic;` if italics is required for styling purposes.
```
Comment (01zulfi) [inline html_css/intermediate_css/more-text-styles.md:75]: The codeblocks are not showing up as intended on the [lesson preview tool](https://www.theodinproject.com/lessons/preview). Adding a new line just before the codeblock should fix it. Can you add that please?

File: html_css/intermediate_css/more-text-styles.md [hunk 4]
```diff
@@ -68,14 +68,24 @@ This method _may_ be more reliable than relying on an third-party font API, but
...
 ~~~css
 h1 {
   font-style: italic;
 }
 ~~~
 
+We should use the `em` element if italics is required for emphasis.
+~~~html
```
Comment (01zulfi) [inline html_css/intermediate_css/more-text-styles.md:83]: Same here as well.

Comment (01zulfi) [inline html_css/intermediate_css/more-text-styles.md:83]: Same here as well.

**Review by 01zulfi — CHANGES_REQUESTED** [review]

**Comment by ItsKatato** [comment]
Oh wow, that's strange! I tested it with the preview tool the first time but I didn't the second cause I thought it was an easy fix. My mistake, I'll remember it next time!

**Review by 01zulfi — APPROVED** [review]
Looks great :rocket:

Thanks for the first time contribution!

**Pull request merged**